script.bash

```diff
@@ -31,33 +31,40 @@ createDirIfNotExist() {
 
 copyConfigurationIfNotExist() {
     if [ ! -s ${DATA_DIR}/${CONFIG} ]; then
-        echo -e "${RED}config.ini file doesn't exist (or it is empty)${DEFAULT}"
+        echo -e "${RED}${CONFIG} file doesn't exist or it is empty. ${GREEN}Creating${DEFAULT}"
```

Comment:
`echo -e "${RED}${CONFIG} file doesn't exist or it is empty. ${GREEN}So creating the file...${DEFAULT}"`

Three dots as we have. :D